tests/policyrep/selinuxpolicy.conf

```diff
@@ -1650,6 +1650,10 @@ type_transition type95 type94:infoflow type95 "name95";
 type_transition type96 type95:infoflow type96 "name96";
 type_transition type97 type96:infoflow type97 "name97";
 
+type_transition type2 type96:infoflow type4 "name97";
+type_transition type3 type96:infoflow type4 "name97";
+type_transition type4 type96:infoflow type97 "name97";
```

Comment:
Why is `"name97"` repeated instead of being incremented, like the lines above?

Reply:
The intent is to have multiple rules with the same key under the new representation so that there are > 1 datums in that key's datum list (there is a separate datum entry for each `otype`) and also multiple `stype`s in at least one datum entry.

But your question made me realize that there should be a comment explaining that :)

Reply:
I'm not clear what you're trying to test. This policy is meant to test that SETools is counting the number of rules correctly, by using prime number amounts of rules to make sure it's counting the right things. 100 doesn't exactly work as a prime :)

Reply:
Hah, I didn't realize the numbers had some [special meaning](https://www.youtube.com/watch?v=QiVfqVQ5t9A) :) Well, I guess I could just remove three existing rules to make room for the new ones...

Anyway, I'm trying to test exactly that - that setools counts the number correctly even in the now "special" cases. The counting function gets more complex with the patch, so I'm just trying to keep the new corner cases covered.